docs/semgrep-cloud-platform/user-management.md

```diff
@@ -155,6 +151,18 @@ To **change a user's role**:
 You cannot change your own role.
 :::
 
+## Setting a default role
+
+Organizations start with a default role of `admin`
+
+To change this, perform the following steps:
+
+1. On Semgrep Cloud Platform's sidebar, click Settings.
+2. On the topbar, click Access.
```

Comment:
```suggestion
2. Click **Access > Defaults**.
```
We can use angle brackets for small steps :) (see [Multi-action procedures](https://developers.google.com/style/procedures#multi-action-procedures)